### ISSUE TITLE
Create YB Slides

### ISSUE THREAD

**Issue by bbengfort** [DistrictDataLabs/yellowbrick#105]
Create a slide presentation that introduces yellowbrick and discusses the concept of a visualizer and how to use it. 

- [x] Presentation Draft 
- [ ] Presentation Review 
- [ ] Post on SlideShare

**Comment by rebeccabilbro** [comment]
Outline

1. An Introduction to Developing Visualizers
2. The Model Selection Process 
    a. YB is part of the model selection workflow  
    b. Feature Selection 
    c. Algorithm Selection (Model Family/Form ⟶ Fitted Model)
    d. Hyperparameter Tuning 
    e. Selection with Cross-Validation 
    f. Try all the models! 
    g. Hyperparameter Tuning 
    h. Model selection is Search 
3. Main Point: Visual Steering Improves Model Selection:
    a. Leads to better models (better F1/R2 Scores)
    b. Gets to good models more quickly 
    c. Produces more insight about models 
4. The Scikit-Learn API 
    a. YB _extends_ the Scikit-Learn API 
    b. Estimators
    c. Transformers 
    d. Visualizers
    e. Pipelines 
    f. Visual Workflows and Pipelines 
5. Primary YB Requirements 
    a. Fits into the sklearn API and workflow 
    b. Implements matplotlib calls efficiently
    c. Low overhead if `poof()` is not called 
    d. Just flexible enough for users to adapt to their data 
    e. Easy to add new visualizers 
    f. Looks as good as Seaborn 
    g. Minimal dependencies: sklearn, numpy, matplotlib -- c'est tout! 
    h. Primary Requirement: Implements Visual Steering 
6. The Visualizer 
    a. Current class hierarchy 
    b. The Visualizer Interface 
    c. Axes management 
    d. `draw()`, `poof()`, and `finalize()`
    e. A simple example of a visualizer 
    f. Feature Visualizers 
    g. Score Visualizers 
    h. Multi-Estimator Visualizers 
9. Visual Pipelines 
    a. Multiple Visualizations 
    b. Interactivity 
10. Optimizing Visualization! 
11. Utilities 
    a. Style management: Sequences, Palettes, Color Codes 
    b. Best Fit Lines 
    c. Type Detection (is_classifier, etc.)
    d. Exceptions     
12. Documentation 
13. User Testing and Research

**Comment by rebeccabilbro** [comment]
[MVP](https://docs.google.com/a/bengfort.com/presentation/d/1ylEDmCVkdWfUYVGx6f0VkQZR2VBiqGcGIRTDlTivib0/edit?usp=sharing)

**Comment by rebeccabilbro** [comment]
Updated outline, per @bbengfort 

1. An Introduction to Developing Visualizers
    a. Yelllowbrick: Model Visualization not Data Visualization 
    b. Goal is not to replace other libraries 
2. The Model Selection Process 
    a. Data science pipeline w/ model + viz in same box (small but important change)
    b. YB is part of the model selection workflow  
    b. Feature Selection 
    c. Algorithm Selection (Model Family/Form ⟶ Fitted Model)
    d. Hyperparameter Tuning 
    e. Selection with Cross-Validation 
    f. Try all the models! 
    g. Hyperparameter Tuning 
    h. Model selection is Search 
3. Main Point: Visual Steering Improves Model Selection:
    a. Leads to better models (better F1/R2 Scores)
    b. Gets to good models more quickly 
    c. Produces more insight about models 
    d. Research People: prove this. 
4. The Scikit-Learn API 
    a. YB _extends_ the Scikit-Learn API 
    b. Tricky because: functional/proceedural matplotlib + OO 
    b. Estimators
    c. Transformers 
    d. Visualizers
    e. Pipelines 
    f. Visual Workflows and Pipelines 
5. Primary YB Requirements 
    a. Fits into the sklearn API and workflow 
    b. Implements matplotlib calls efficiently
    c. Low overhead if `poof()` is not called 
    d. Just flexible enough for users to adapt to their data 
    e. Easy to add new visualizers 
    f. Looks as good as Seaborn 
    g. Minimal dependencies: sklearn, numpy, matplotlib -- c'est tout! 
    h. Primary Requirement: Implements Visual Steering 
6. The Visualizer 
    a. Current class hierarchy 
    b. The Visualizer Interface 
    c. Axes management 
    d. `draw()`, `poof()`, and `finalize()`
    e. A simple example of a visualizer 
    f. Feature Visualizers 
    g. Score Visualizers 
    h. Multi-Estimator Visualizers 
9. Visual Pipelines 
    a. Multiple Visualizations 
    b. Interactivity 
10. Optimizing Visualization! 
11. Utilities 
    a. Style management: Sequences, Palettes, Color Codes 
    b. Best Fit Lines 
    c. Type Detection (is_classifier, etc.)
    d. Exceptions     
12. Documentation and Sphinx
13. Contributing 
    a. Git/Branch Managment 
    b. Issues, milestones, and labels 
    c. Waffle Board 
13. User Testing and Research
